themes/theme-b2b/src/index.ts

```diff
@@ -548,6 +548,10 @@ const theme: BaseTheme = {
         color: colors.gray00,
         bg: colors.blue60,
       },
+      disabled: {
+        ...selectOption,
+        color: colors.gray40,
```

Comment:
maybe add a cursor: 'not-allowed' as we use this in other `disabled` components aswell